web/source/build.sh

```diff
@@ -307,33 +314,27 @@ if [ $DO_MINIFY = true ]; then
         exit 1
     fi
 
-    # Also, build our sourcemap-root tool for cleaning up the minified version's sourcemaps.
-    echo "Compiling build tools for minified build products"
-    $compiler --build "source/$minified_sourcemap_cleaner/tsconfig.json"
-    assert "$minified_sourcemap_cleaner/index.js"
-fi
-
-if [ $BUILD_LMLAYER = true ]; then
-    # Ensure that the LMLayer compiles properly, readying the build product for comsumption by KMW.
-    cd ../../common/predictive-text/
-    echo ""
-    echo "Compiling the Language Modeling layer module..."
-    ./build.sh || fail "Failed to compile the language modeling layer module."
-    cd ../../web/source
-    echo "Copying ${PREDICTIVE_TEXT_SOURCE} to ${PREDICTIVE_TEXT_OUTPUT}"
-    cp "${PREDICTIVE_TEXT_SOURCE}" "${PREDICTIVE_TEXT_OUTPUT}" || fail "Failed to copy predictive text model"
-    echo "Language Modeling layer compilation successful."
-    echo ""
+    if [ $FETCH_DEPS = true ]; then
+        # Also, build our sourcemap-root tool for cleaning up the minified version's sourcemaps.
+        echo "Compiling build tools for minified build products"
+        $compiler --build "source/$minified_sourcemap_cleaner/tsconfig.json"
+        assert "$minified_sourcemap_cleaner/index.js"
+    fi
 fi
 
 if [ $BUILD_CORE = true ]; then
-    # Ensure that the KeyboardProcessor module compiles properly.
-    cd ../../common/core/web/keyboard-processor/src
+    CORE_FLAGS="-no-lerna"
+    if [ $BUILD_LMLAYER = false ]; then
+        CORE_FLAGS="$CORE_FLAGS -test"
+    fi
```

Comment:
One change I'd like to point out:  this new `-no-lerna` option I've added to a number of the build scripts.

Use of `lerna bootstrap` automatically runs `npm install` on all `lerna`-managed packages, so there's no need to run the command multiple times.  We've noted that it might be nice to skip this check (when it was `npm install`) on occasion in the past, which was also part of the motivation.

I'm pretty sure we want this as an option; it does help keep the build logs a bit cleaner.  That said... I'm not exactly sold on `-no-lerna` as the option's name.  Any preferences for something better?

Ideas:
- `-skip-deps` / `-skip-dependencies`
    - Caveat:  this may imply skipping a re-build of `keyboard-processor` when building KMW.  Might be good to (_also_) have, but it's semantically distinct.  To me, `skip-deps` sounds close to one aspect of KMW's `-test` option.
        - While `-test` does skip the embedded and UI builds, the notable aspect (for this conversation) is that it skips `lm-layer` re-builds.
- `-skip-package-install`
    - Is more verbose, but probably clearer.

Reply:
`-skip-package-install` :+1: (can always have `-S` as shorthand documented in the script help.